tests/foreman/ui/test_registration.py

```diff
@@ -0,0 +1,337 @@
+            **{f'rhel{rhelver}_os': settings.repos[f'rhel{rhelver}_os']}
+        )
+    # make sure there will be package availabe for update
+    if rhel_contenthost.os_version.major == '6':
+        package = FAKE_1_CUSTOM_PACKAGE
+        repo_url = settings.repos.yum_1['url']
+    else:
+        package = FAKE_7_CUSTOM_PACKAGE
+        repo_url = settings.repos.yum_3['url']
+    rhel_contenthost.create_custom_repos(fake_yum=repo_url)
+    rhel_contenthost.execute(f"yum install -y {package}")
+    # run curl
+    result = rhel_contenthost.execute(cmd)
+    assert result.status == 0
+    result = rhel_contenthost.execute('subscription-manager identity')
+    assert result.status == 0
+    # Assert that a yum update was made this day ("Update" or "I, U" in history)
+    timezone_offset = rhel_contenthost.execute('date +"%:z"').stdout.strip()
+    tzinfo = datetime.strptime(timezone_offset, '%z').tzinfo
+    result = rhel_contenthost.execute('yum history | grep U')
+    assert result.status == 0
+    assert datetime.now(tzinfo).strftime('%Y-%m-%d') in result.stdout
+    # Set "Connect to host using IP address"
+    target_sat.api.Parameter(
+        host=rhel_contenthost.hostname,
+        name='remote_execution_connect_by_ip',
+        parameter_type='boolean',
+        value='True',
+    ).create()
+    # run insights-client via REX
+    command = "insights-client --status"
+    invocation_command = target_sat.cli_factory.make_job_invocation(
+        {
+            'job-template': 'Run Command - Script Default',
+            'inputs': f'command={command}',
+            'search-query': f"name ~ {rhel_contenthost.hostname}",
+        }
+    )
+    # results provide all info but job invocation might not be finished yet
+    result = (
+        target_sat.api.JobInvocation()
+        .search(
+            query={'search': f'id={invocation_command["id"]} and host={rhel_contenthost.hostname}'}
+        )[0]
+        .read()
+    )
+    # make sure that task is finished
+    task_result = target_sat.wait_for_tasks(
+        search_query=(f'id = {result.task.id}'), search_rate=2, max_tries=60
+    )
+    assert task_result[0].result == 'success'
+    host = (
+        target_sat.api.Host()
+        .search(query={'search': f'name={rhel_contenthost.hostname}'})[0]
+        .read()
+    )
+    for interface in host.interface:
+        interface_result = target_sat.api.Interface(host=host.id).search(
+            query={'search': f'{interface.id}'}
+        )[0]
+        # more interfaces can be inside the host
+        if interface_result.identifier == iface:
+            assert interface_result.execution
+
+
+@pytest.mark.tier2
+def test_global_registration_form_populate(
+    module_org,
+    session,
+    module_ak_with_cv,
+    module_lce,
+    module_promoted_cv,
+    default_architecture,
+    default_os,
+    target_sat,
+):
+    """Host registration form should be populated automatically based on the host-group
```

Comment:
I would use `host group` or `Host Group` instead of `host-group` since they are more established in our test framework and as well as Satellite.

Reply:
I don't understand. Can you please elaborate?

Reply:
never mind, let's skip this one. I just wanted you to remove the hyphen from `host-group`.